src/components/Chef.js

```diff
@@ -0,0 +1,96 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import { Link as LinkComponent } from 'react-router';
+import actions from '../action_creators';
+import constants from '../constants';
+
+const Link = createFactory(LinkComponent);
+const displayMax = 6;
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+function toOpenEntries(orderId, entry, entryIndex) {
```

Comment:
What do you think of this data structure? I couldn't think of any other way for the open entry to know which order, and index in that order it belongs to. Did it this way so I can reuse the changeEntryStatus action.